[
  {
    "@timestamp": "2020-03-31T07:50:27.1681182Z",
    "event": {
      "action": "kerberos-preauth-failed",
      "category": "authentication",
      "code": 4771,
      "kind": "event",
      "module": "security",
      "outcome": "failure",
      "provider": "Microsoft-Windows-Security-Auditing",
      "type": "start"
    },
    "host": {
      "name": "DC_TEST2k12.TEST.SAAS"
    },
    "log": {
      "level": "information"
    },
    "related": {
      "user": "MPUIG"
    },
    "source": {
      "ip": "192.168.5.44",
      "port": 53366
    },
    "user": {
      "name": "MPUIG"
    },
    "winlog": {
      "api": "wineventlog",
      "channel": "Security",
      "computer_name": "DC_TEST2k12.TEST.SAAS",
      "event_data": {
        "PreAuthType": "0",
        "ServiceName": "krbtgt/test.saas",
        "Status": "0x12",
        "StatusDescription": "KDC_ERR_CLIENT_REVOKED",
        "TargetSid": "S-1-5-21-1717121054-434620538-60925301-3057",
        "TargetUserName": "MPUIG",
        "TicketOptions": "0x40810010",
        "TicketOptionsDescription": [
          "Renewable-ok",
          "Name-canonicalize",
          "Renewable",
          "Forwardable"
        ]
      },
      "event_id": 4771,
      "keywords": [
        "Audit Failure"
      ],
      "opcode": "Info",
      "process": {
        "pid": 496,
        "thread": {
          "id": 4552
        }
      },
      "provider_guid": "{54849625-5478-4994-a5ba-3e3b0328c30d}",
      "provider_name": "Microsoft-Windows-Security-Auditing",
      "record_id": 5027836,
      "task": "Kerberos Authentication Service"
    }
  }
]